Rachelzitomer patch 2

rachelzitomer opened the pull request:
Added some additional files and made a small update to bomcal.north data.

rachelzitomer committed:
multicore demo

added a brief demo of bootstraps using multicore processing at end of fit_stand_age_and_roads.R

rachelzitomer committed:
Create GEE_script.txt

added GEE_script.txt to scripts-this is the Google Earth Engine code used to generate original stand age rasters and download NLCD data

rachelzitomer committed:
structural connectivity analysis

added structural_connectivity.R to scripts and added the following associated data files (folder in parentheses):
colony_abundance.csv (raw_data)
midnorth_early_seral_binary_map.tif (raw_data)
north_early_seral_binary_map.tif (raw_data)
south_early_seral_binary_map.tif (raw_data)
midnorth_latlongs.csv (raw_data)
north_latlongs.csv (raw_data)
south_latlongs.csv (raw_data)
structural_connectivity.RData (data)

rachelzitomer committed:
updated north.bomcal data

updated north.bomcal data to remove a queen that wasn't used in final analysis (both raw data files and .RData)

nspope commented on inst/raw_data/desktop.ini:
```diff
@@ -0,0 +1,2 @@
+[LocalizedFileNames]
```
Is this file needed?

rachelzitomer replied:
No, good catch. I've deleted it.

nspope reviewed (commented):
Looks good -- could you please add a README.md to the "inst" directory that gives a brief description of what the files are?

rachelzitomer committed:
Pull request edits

deleted unnecessary desktop.ini file generated for colony_abundance.csv

added oc_age_canopycover.csv (data from 2018-2019 study used to project canopy cover from stand age)

renamed north.roads.R -> midnorth.roads.R (name appeared to be in error)

Added Readme.md file for inst

rachelzitomer commented:
> Looks good -- could you please add a README.md to the "inst" directory that gives a brief description of what the files are?

I added a Readme.md in the inst directory, can you take a look at your file descriptions for your files (particularly those in scripts) and make any changes needed?
